04 home page

iliajamasb opened the pull request:
Added nav bar, clearer home page, and a placeholder request table (though the layout can be considered for later).

Also implemented the tests for the nav bar redirects

iliajamasb committed:
Created and implemented navbar

iliajamasb committed:
Added home, navbar and bookings page

Ali-Alkhars commented on lessons/templates/partials/navbar.html:
```diff
@@ -0,0 +1,27 @@
+        <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav">
+                <li class="nav-item active">
+                    <a class="nav-link" href='{% url 'home' %}'>Home</a>
```
This is kinda redundant cause the logo already does the same thing, so you could just delete this

Ali-Alkhars commented on lessons/templates/partials/navbar.html:
```diff
@@ -0,0 +1,27 @@
+                    <a class="nav-link" href='{% url 'home' %}'>Home</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href='{% url 'lesson_requests' %}'>Bookings</a>
```
I guess here it was a typo, the URL should be 'bookings' instead of lesson_requests

Ali-Alkhars reviewed (changes requested):
I think what you did is pretty good for now. Just edit what I pointed out in the other comments then I'll solve the conflicts and merge.

Just to keep in mind, more buttons will be added to the nav-bar later and the style of the nav-bar should be enhanced in your new UI ticket.

Well done overall!

iliajamasb committed:
Updated navbar

iliajamasb committed:
Merge branch 'main' into 04-home-page

iliajamasb committed:
Improved navbar and home page

iliajamasb committed:
Corrected erroneous test

Ali-Alkhars commented on lessons/views.py:
```diff
@@ -69,7 +69,7 @@ def log_out(request):
     return redirect('main')
 
-@login_required
+#@login_required
```
uncomment this

Ali-Alkhars reviewed (approved):
Well done Ilia! I really like the new background picture.

I have approved this request but please uncomment the login_required decorator I pointed out and push then merge the branch.

iliajamasb committed:
Corrected log_out bug

Had to uncomment a line which now correctly requires user to be logged in to access bookings page
